webapp/security/views.py

```diff
@@ -41,13 +55,14 @@ def notice(notice_id):
     package_versions = {}
     release_packages = SortedDict()
 
+    all_releases = security_api.get_releases().get("releases")
+
     releases = {
-        release.codename: release.version
-        for release in db_session.query(Release).all()
+        release["codename"]: release["version"] for release in all_releases
     }
 
-    if notice.release_packages:
-        for codename, pkgs in notice.release_packages.items():
+    if notice["release_packages"]:
+        for codename, pkgs in notice["release_packages"].items():
             release_version = releases[codename]
```

Comment:
Hmm. Looking into this, I don't think you need the `releases` endpoint at all.

So, we only need the `releases` data for getting the `release_version` for every release codename mentioned in `release_packages`.

It looks to me like every notice JSON response (e.g. https://ubuntu.com/security/notices/USN-5340-1.json or https://ubuntu.com/security/notices/LSN-0085-1.json) already has a `releases` property, which contains exactly the set of releases mentioned in the `release_packages` list.

So we really just need to do the dictionary you're making in line 44 from `notice["releases"]` rather than `all_releases` and that should be it.